force-app/main/default/objects/Medication/recordTypes/EDRD.recordType-meta.xml

```diff
@@ -106,10 +198,6 @@
             <fullName>Active</fullName>
             <default>true</default>
         </values>
-        <values>
-            <fullName>Added in error</fullName>
```

Comment:
@deepakmulamalla Why is this removed? was this business requirement?

Reply:
Yes @cgiSunilM, It is business requirement to have only two values, i.e., Active & Inactive.